src/pages/api/cancel.ts

```diff
@@ -1,12 +1,14 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 
-const BASE_URL = process.env.NEXT_PUBLIC_SEQERA_API_URL;
-const WORKSPACE_ID = process.env.NEXT_PUBLIC_WORKSPACE_ID;
-const token = process.env.NEXT_PUBLIC_SEQERA_ACCESS_TOKEN!;
-
+const BASE_URL = process.env.SEQERA_API_URL;
+const WORKSPACE_ID = process.env.WORKSPACE_ID;
+const token = process.env.SEQERA_ACCESS_TOKEN!;
```

Comment:
Is this working OK only having the access token accessible on the server side? That's great if it works without further changes.